gdk/src/keys.rs

```diff
@@ -44,6 +45,18 @@ impl IntoGlib for Key {
 }
 
 impl Key {
+    #[doc(alias = "gdk_keyval_from_name")]
+    pub fn from_name(keyval_name: &str) -> Self {
```

Comment:
This can't fail?

Reply:
The documentation says it returns "`GDK_KEY_VoidSymbol` if the key name is not a valid key". So I guess not a failure? This is different from `gdk_keymap_lookup_key` which returns `0` if no mapping exists.

Reply:
Strange API, but ok :)